crystal-map-connector/src/main/kotlin/ch/ergon/dope/extension/type/ObjectField.kt

```diff
@@ -0,0 +1,52 @@
+package ch.ergon.dope.extension.type
+
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.toDopeType
+import ch.ergon.dope.validtype.ObjectType
+import com.schwarz.crystalapi.schema.CMJsonField
+import com.schwarz.crystalapi.schema.CMJsonList
+import com.schwarz.crystalapi.schema.CMObjectField
+import com.schwarz.crystalapi.schema.CMObjectList
+import com.schwarz.crystalapi.schema.CMType
+import com.schwarz.crystalapi.schema.Schema
+import kotlin.reflect.KProperty1
+
+class ObjectField<S : Schema>(val schema: S, val name: String, val path: String) : Field<ObjectType>(name, path)
+
+inline fun <reified T : CMType, S : Schema> ObjectField<S>.getField(field: KProperty1<S, T>): T {
+    val schemaField = field.get(schema)
+    val newPath = if (path.isBlank()) name else "$path`.`$name"
```

Comment:
Why are you only adding ` after path and before name? That looks quite hacky

Reply:
moved it into the `toDopeQuery()` of the field